corehq/apps/data_interfaces/const.py

```diff
@@ -0,0 +1,6 @@
+from django.utils.translation import gettext_lazy
+
+CRITERIA_OPERATOR_CHOICES = [
+    ('ALL', gettext_lazy('ALL of the criteria are met')),
+    ('ANY', gettext_lazy('ANY of the criteria are met')),
+]
```

Comment:
FYI in Django 3 we have access to a nice `Enum` type for model field choices.  I just switched a similar bit of code to use it here if you want to see what it looks like:
https://github.com/dimagi/commcare-hq/pull/31476
That makes it real explicit where `"ALL"` comes from when referenced elsewhere.

Reply:
I think I'll be able to make this change. It doesn't look like your change required a migration?

Reply:
Looks like it works fine without re-running the migration! (which makes sense, because the data is the exact same) Added!

Reply:
nice, thanks!